utils/generate_types/overrides-test.d.ts

```diff
@@ -66,7 +66,7 @@ export interface Config<TestArgs = {}, WorkerArgs = {}> extends TestConfig {
   use?: UseOptions<TestArgs, WorkerArgs>;
 }
 
-export type Metadata = { [key: string]: string | number | boolean };
+export type Metadata = { [key: string]: any };
```

Comment:
Out of curiosity, why not

```suggestion
export type Metadata = Record<string, any>;
```

Reply:
Stylistic pref+habit. We use a mix internally in overrides-test.d.ts.

(Some `Record`, some `{…}` variants. I find for simple types, not using parameterized helpers easier to read. When I was new to TS, if I saw `{ [key: string]: any }` I knew exactly what it meant.  When I saw `Record<string, any>`, I was like…"What's this Record type? I didn't know TS has Record. Is this different than an object?!")